.envrc

```diff
@@ -0,0 +1,8 @@
+NODEVERSION=16
```

Comment:
```suggestion
NODEVERSION=18
```

Reply:
Done. I used version `16`  because that is what you currently use in the workflow.

https://github.com/chaiNNer-org/chaiNNer/blob/877d03249e1a9a1179b5cf0fd8e4557412a223c1/.github/workflows/release.yml#L33